### PR TITLE
Reorganizing intro material

### PR DESCRIPTION
**Pull request by ChristinaLK**
@sstevens2 sparked my interest in this material (I'll be teaching it in April) and I did a rewrite of the opening. :) More changes/suggestions to come. 

I did cut a lot of the more technical details (maybe too much?) so feedback is welcome. 

~Also, don't merge yet, I want to add at least one more analogy Sarah shared about images vs containers!~

### CODE REVIEW

**Comment by ChristinaLK** [comment]
calling for review from @dme26 @sstevens2 !

at your leisure. ;)

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -28,74 +27,138 @@ This lesson comes with a whole set of disclaimers; my apologies for needing them
...
+files. These all 
+live in what's called the file system. Everything on your computer - programs, 
+pictures, documents - lives somewhere in the file system. One way to think of 
+the file system is the layer of stuff that can be activated to use use the CPU, memory and hard 
+drive of your computer. 
+
+NOW, imagine you wanted to have a second computer. You don't want to buy a 
+whole new computer because it's too expensive. What if, instead, you could have 
+another filesystem that you could store and access from your main computer, 
+but that is self-container?
```
Comment (sstevens2) [inline _episodes/01-introduction.md]: self-contained

**Comment by sstevens2** [comment]
I really like this re-write.  I would be happy to merge.  @dme26 do you have any feedback?

**Comment by sstevens2** [comment]
If we don't hear from @dme26 by Feb 10th, I'll merge this.

**Pull request merged**